src/java.base/share/classes/sun/net/www/http/HttpClient.java

```diff
@@ -165,13 +175,25 @@ int getKeepAliveTimeout() {
         return keepAliveTimeout;
     }
 
+    static String normalizeCBT(String s) {
+        if (s == null || ! (s.equals("always") ||
+                s.equals("never") || s.startsWith("domain:"))) {
```

Comment:
I guess there's a `!` missing in front of  `s.startsWith("domain:")` here?

Reply:
Argh - you're right I missed the fact that the 3 expressions where included in parenthesis. I read it as 
```
! (s.equals("always")) || ...
```

Reply:
Shall we log a message if the value is not one of the 3 forms?

Reply:
Usually malformed values are just ignored - and the property takes its default value. But yes - s.n.w.h.HttpClient has a logger so it wouldn't be much effort to log it as a DEBUG trace for better diagnostic.

Reply:
Yes, I will log it using the same debug/logging mechanism already in the same source file..